pages/history.js

```diff
@@ -1,10 +1,92 @@
+import {
+	faBirthdayCake,
+	faCircleNotch,
+	faMask,
+	faRandom,
+	faVideo,
+} from "@fortawesome/free-solid-svg-icons";
 import Head from "next/head";
 import styled from "styled-components";
+import Event from "../components/event";
+
+const EventsContainer = styled.div`
+	display: flex;
+	flex-wrap: wrap;
+	justify-content: center;
+
+	& > * {
+		flex: 0 1 calc(50% - 16px);
+		margin: 8px;
+
+		max-width: 35%;
+	}
+
+	@media (max-width: 800px) {
+		flex-direction: column;
+
+		& > * {
+			flex-basis: 100%;
+			max-width: unset;
+		}
+	}
+`;
 
 export default () => (
 	<>
 		<Head>
 			<title>History - The Snakeroom</title>
 		</Head>
+		<EventsContainer>
+			<Event icon={faMask} name="Imposter" time="April 2020">
+				In this event, players receive four messages by other players
+				and one from the bot, and must determine which one was generated
+				by the bot. The third iteration of the Sneknet collects results
+				from each round to determine which of the five messages was
+				generated by the bot. With enough data, the correct answer can
+				automatically be selected.
+			</Event>
+			<Event icon={faVideo} name="RPAN" time="August 2019">
+				RPAN is a streaming event with a cap on streamers and length.
+				The Snakeroom used a script that sniped slots and allowed users
+				to broadcast from an OBS stream, bypassing the intended
+				limitation of streaming from a mobile device and allowing
+				Snakeroom members to create <b>Place Live</b> and{" "}
+				<b>RPAN Plays Pokémon</b>. They were also able to stream prior
+				to the event going live, although Reddit admins quickly shut
+				this down.
+			</Event>
+			<Event icon={faRandom} name="Sequence" time="April 2019">
+				During this event, players were able to upload and vote on GIFs,
+				with the highest-upvoted GIFs selected to become part of the
+				final video. The second iteration of the Sneknet locks in select
+				GIFs using the power of Snakeroom participants, without chancing
+				success on the whims of the wider Reddit community.
```

Comment:
```suggestion
				During this event, players are able to upload and vote on GIFs,
				with the highest-upvoted GIFs selected to become part of the
				final video. The second iteration of the Sneknet locks in selected
				GIFs using the power of Snakeroom participants, without chancing
				success on the whims of the wider Reddit community.
```

Reply:
Resolved by instead changing to past tense in 74b3de0122571f988d2b292819c1e11bf2474a15